fakebook/src/app/service/upload-image.service.ts

```diff
@@ -0,0 +1,27 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { OktaAuthService } from '@okta/okta-angular';
+import { environment } from 'src/environments/environment';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class UploadImageService {
+  baseUrl = `${environment.profile.url}/api/profiles/`;
+
+  constructor(private http: HttpClient, private oktaAuth: OktaAuthService) { }
+
+  public UploadImage(formData: FormData): Promise<any> {
```

Comment:
naming convention. and you should be able to use a more specific type than `any`, presumably `{ path: string, userId: number }`, which could also be an interface.